2020-07-31T15:01:02.6606232+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T15:01:02.6941858+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T15:01:02.6945908+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T15:11:00.9483828+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T15:11:00.9579430+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T15:11:00.9580991+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T15:22:54.3015520+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T15:22:54.3122262+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T15:22:54.3124018+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T15:28:20.3093365+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T15:28:20.3218034+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T15:28:20.3220100+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T15:30:19.2235203+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T15:30:19.2360100+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T15:30:19.2361514+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T15:40:52.8321152+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T15:40:52.8424638+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T15:40:52.8426630+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T15:43:17.8326489+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T15:43:17.8423296+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T15:43:17.8424990+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T15:45:27.5254946+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T15:45:27.5348547+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T15:45:27.5352724+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T15:47:21.0939387+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T15:47:21.1020845+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T15:47:21.1022599+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T15:47:49.7592513+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T15:47:49.7706530+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T15:47:49.7708185+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T15:48:30.9473797+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T15:48:30.9567334+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T15:48:30.9569193+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T16:03:57.9203144+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T16:03:57.9290239+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T16:03:57.9292943+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T16:05:02.7793622+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T16:05:02.7880336+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T16:05:02.7881698+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T16:05:56.2430649+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T16:05:56.2535417+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T16:05:56.2537367+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T16:22:20.7106933+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T16:22:20.7211113+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T16:22:20.7212776+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T16:22:56.7292128+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T16:22:56.7399797+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T16:22:56.7401510+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T16:24:06.1177713+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T16:24:06.1298203+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T16:24:06.1300536+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T16:31:15.5625229+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T16:31:15.5741212+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T16:31:15.5765850+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T16:33:32.5853945+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T16:33:32.6347125+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T16:33:32.6349055+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T16:33:59.4983346+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T16:33:59.5091857+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T16:33:59.5095545+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T16:57:19.2034996+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-07-31T16:57:19.2147751+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-07-31T16:57:19.2149509+07:00  [INF] Content root path: "D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web" (b5d60022)
2020-07-31T16:57:26.3228097+07:00 80000019-0003-fe00-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.AggregateException: One or more errors occurred. (No connection could be made because the target machine actively refused it.)
 ---> System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.
 ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at System.Threading.Tasks.Task`1.get_Result()
   at SubcontractProfile.Web.Controllers.AccountController.DDLsubcontract_profile_province() in D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web\Controllers\AccountController.cs:line 416
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-07-31T16:57:26.3228095+07:00 8000001d-0001-f800-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.AggregateException: One or more errors occurred. (No connection could be made because the target machine actively refused it.)
 ---> System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.
 ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at System.Threading.Tasks.Task`1.get_Result()
   at SubcontractProfile.Web.Controllers.AccountController.DDLsubcontract_profile_sub_district(Int32 district_id) in D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web\Controllers\AccountController.cs:line 300
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-07-31T16:57:26.3336421+07:00 80000018-0001-f300-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.AggregateException: One or more errors occurred. (No connection could be made because the target machine actively refused it.)
 ---> System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.
 ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at System.Threading.Tasks.Task`1.get_Result()
   at SubcontractProfile.Web.Controllers.AccountController.DDLsubcontract_profile_district(Int32 province_id) in D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web\Controllers\AccountController.cs:line 377
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-07-31T16:57:26.3498720+07:00 8000001a-0000-f400-b63f-84710c7967bb [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.AggregateException: One or more errors occurred. (No connection could be made because the target machine actively refused it.)
 ---> System.Net.Http.HttpRequestException: No connection could be made because the target machine actively refused it.
 ---> System.Net.Sockets.SocketException (10061): No connection could be made because the target machine actively refused it.
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task`1.GetResultCore(Boolean waitCompletionNotification)
   at System.Threading.Tasks.Task`1.get_Result()
   at SubcontractProfile.Web.Controllers.AccountController.DDLTitle() in D:\Developer Project\SubcontractProfile\Presentation\Web\SubcontractProfile.Web\Controllers\AccountController.cs:line 437
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
